central/convert/storagetov2/vulnerability.go

```diff
@@ -20,3 +20,106 @@ func convertVulnerabilityState(state storage.VulnerabilityState) v2.Vulnerabilit
 		return v2.VulnerabilityState_OBSERVED
 	}
 }
+
+func EmbeddedVulnerabilities(vulns []*storage.EmbeddedVulnerability) []*v2.EmbeddedVulnerability {
+	if len(vulns) == 0 {
+		return nil
+	}
+
+	var ret []*v2.EmbeddedVulnerability
+	for _, vuln := range vulns {
+		if vuln == nil {
+			continue
+		}
+		ret = append(ret, EmbeddedVulnerability(vuln))
+	}
+
+	return ret
+}
+
+func EmbeddedVulnerability(vuln *storage.EmbeddedVulnerability) *v2.EmbeddedVulnerability {
+	if vuln == nil {
+		return nil
+	}
+
+	result := &v2.EmbeddedVulnerability{
+		Cve:               vuln.GetCve(),
+		Summary:           vuln.GetSummary(),
+		Link:              vuln.GetLink(),
+		VulnerabilityType: convertVulnerabilityType(vuln.GetVulnerabilityType()),
+		Severity:          convertSeverity(vuln.GetSeverity()),
+		CvssV3:            CvssV3(vuln.GetCvssV3()),
+		PublishedOn:       vuln.GetPublishedOn(),
+		LastModified:      vuln.GetLastModified(),
+		// ScoreVersion: convertScoreVersion(vuln.GetScoreVersion()),
```

Comment:
uncomment, delete, or add a TODO referencing a ticket so we can find this easily.

Reply:
https://issues.redhat.com/browse/ROX-26066 says this can be deprecated, so I'm removing it.